cmd/argo/commands/cron/get.go

```diff
@@ -73,7 +73,7 @@ func getCronWorkflowGet(cwf *wfv1.CronWorkflow) string {
 	out += fmt.Sprintf(fmtStr, "Name:", cwf.ObjectMeta.Name)
 	out += fmt.Sprintf(fmtStr, "Namespace:", cwf.ObjectMeta.Namespace)
 	out += fmt.Sprintf(fmtStr, "Created:", humanize.Timestamp(cwf.ObjectMeta.CreationTimestamp.Time))
-	out += fmt.Sprintf(fmtStr, "Schedule:", cwf.Spec.Schedule)
+	out += fmt.Sprintf(fmtStr, "Schedule:", cwf.Spec.GetScheduleString())
```

Comment:
```suggestion
	out += fmt.Sprintf(fmtStr, "Schedules:", cwf.Spec.GetScheduleString())
```

similar to above

Reply:
Fixed in #13369 ([this file](https://github.com/argoproj/argo-workflows/pull/13369/files#diff-85b5b6e77fc3dd06d616bb8a44ba63c95f7bc8193c993e7e0ed04cc203ad6aeb))